polaris.shopify.com/content/contributing/working-with-the-polaris-team/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Working with the Polaris team
+description: When you contribute to Polaris, you help the Shopify community create better experiences for merchants, faster. Contribution takes thoughtful planning.
+keywords:
+  - service level agreement
+  - SLA
+  - contributing to polaris
+  - open source
+  - contribution guide
+---
+
+Through supporting teams, we’ve learned what works well, and not so well. We’ve outlined these things so that you can create a successful plan. This guide goes into detail, but here are the highlights.
+
+## TL;dr
+
+Think about your system needs in the prototype phase of your project so your team can plan for [quality](/contributing/quality-contributions) systems [contributions](/contributing) ahead of the build phase. The Polaris team is here to help you with planning, but we can’t be pulled in last minute to unblock. 
+
+The best way to get help is through our support channels:
+- [GitHub discussions](https://github.com/Shopify/polaris/discussions/6750) to talk to the community about anything from patterns, to potential contributions 
+-  #polaris in Slack if you work at Shopify and have a general inquiry
+
+As you’re building you may need custom styles. In those cases, always use Polaris [tokens](/tokens).
+
+## Product team tips
+With some early planning and intentional systems thinking, the build stage will go a lot smoother.
+
+### Plan for systems changes ahead of time
+
+The Polaris team should never be pulled in to unblock a team from shipping something quickly—we’re here to support and guide teams when they’re planning their strategic system changes ahead of time. Teams at Shopify should start working on their system needs early in the design and build process and are expected to make [quality system contributions](link to new quality guidelines) ahead of their ship dates. 
```

Comment:
```suggestion
The Polaris team should never be pulled in to unblock a team from shipping something quickly—we’re here to support and guide teams when they’re planning their strategic system changes ahead of time. Teams at Shopify should start working on their system needs early in the design and build process and are expected to make [quality system contributions](/contributing#what-makes-a-good-contribution) ahead of their ship dates. 
```

Reply:
swapping for the current quality guidelines that @raquelbreternitz and @jjgali shipped, might make sense to swap them out later when we have more quality docs